lib/compact.js

```diff
@@ -51,9 +51,9 @@ module.exports = api;
  * @param options the compaction options.
  * @param compactionMap the compaction map to use.
  *
- * @return the compacted value.
+ * @return a promise that resolves to the compacted value.
  */
-api.compact = ({
+api.compact = async ({
```

Comment:
I assume  this maps directly to `jsonld.compact` in which case this breaking change from sync to async warrants a mention in the changelog.

Reply:
It does not map directly. `jsonld.compact` was already async. The internals were not -- so this is not a public facing change. Same for the other internals.